jstz_cli/src/repl.rs

```diff
@@ -44,7 +44,7 @@ pub fn exec(self_address: Option<String>, cfg: &Config) -> Result<()> {
         },
         rt.context(),
     );
-    realm_clone.register_api(TextEncoderApi, rt.context());
+    realm_clone.register_api(GlobalApi, rt.context());
```

Comment:
`GlobalApi` should be a sub-api of `TextEncoderApi`, that is to say, when registering the text encoder api, we should register the global api for text encoder.

Reply:
we can wait to merge https://github.com/trilitech/jstz/pull/190

Reply:
done